stylecop.ruleset

```diff
@@ -21,6 +21,8 @@
     <Rule Id="SA1311" Action="None" />
     <!-- SA1401FieldsMustBePrivate -->
     <Rule Id="SA1401" Action="None" />
+    <!-- SA1402FileMayOnlyContainASingleType -->
+    <Rule Id="SA1402" Action="None" />
```

Comment:
Do not enable this for easy locating source of a type

Reply:
you mean do not disable?

---
In reply to: [185723267](https://github.com/dotnet/docfx/pull/2717#discussion_r185723267) [](ancestors = 185723267)

Reply:
yes

---
In reply to: [185723428](https://github.com/dotnet/docfx/pull/2717#discussion_r185723428) [](ancestors = 185723428,185723267)

Reply:
enabled and split states and rules

---
In reply to: [185723572](https://github.com/dotnet/docfx/pull/2717#discussion_r185723572) [](ancestors = 185723572,185723428,185723267)